src/styles/elements/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$card-transition-color: #34568b;
+$nav-transition-color: #00acdc;
```

Comment:
These colours can be used for other purposes too, so better to name them as it looks like (grey, blue, etc.) or as what they used for (text-main, text-secondary, etc.)

Reply:
now I start to name colors like you write, $page-color, $button-color or $color-grey, $color-light-red